src/controllers/check.your.answers.controller.ts

```diff
@@ -27,8 +27,9 @@ export const get = (req: Request, res: Response, next: NextFunction) => {
     logger.infoRequest(req, `${config.CHECK_YOUR_ANSWERS_PAGE} hasTrusts=${hasTrusts}`);
 
     let backLinkUrl: string = config.BENEFICIAL_OWNER_TYPE_URL;
+
     if (hasTrusts) {
-      backLinkUrl = config.TRUST_INFO_PAGE;
+      backLinkUrl = `${config.TRUSTS_URL}`;
```

Comment:
I think the back link change is being tracked by a Jira ticket so might be best to make those changes on a seperate PR or see if you can bring that jira into this one. I would double check with Steve